modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -283,39 +282,35 @@ private void purgeService(DeployTask deployTask, DeployServiceEntity deployServi
     /**
      * Deployment service.
      *
-     * @param newId         new service id.
-     * @param userId        user id.
-     * @param deployRequest deploy request.
-     * @return new deployed service entity.
+     * @param processInstanceId ProcessInstance ID
```

Comment:
we do not need processInstanceId.. it must be transparent to the new service deployer.